apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -102,7 +102,10 @@ DropletBlockTooltipManager.prototype.installTooltipsForCurrentCategoryBlocks_ =
       }.bind(this)
     });
 
-    // Store the title/funcName as a block id so we can attach callouts later:
+    // Store the title/funcName as data-block so we can attach callouts later:
+    $(blockHoverDiv).attr('data-block', funcName);
```

Comment:
Should we use [`jQuery.data(key, value)`](https://api.jquery.com/data/#data1) for setting data attributes instead?